Src/CMakeLists.txt

```diff
@@ -39,15 +39,14 @@ if (LIBCZI_BUILD_CURL_BASED_STREAM)
         GIT_TAG "curl-8_4_0"
         # Set the prefix to control where it's installed
         PREFIX "${CMAKE_BINARY_DIR}/vendor/curl"
```

Comment:
This differs from the previous directory <CMAKE_BINARY_DIR>/curl but I am assuming that is intended since I see vendor used for other projects as well